lib/OpenQA/Worker/Jobs.pm

```diff
@@ -249,35 +271,23 @@ sub upload {
         return 0;
     }
 
-    # double check uploads if the webui asks us to
-    if ($res->res->json && $res->res->json->{temporary}) {
-        my $csum1 = '1';
-        my $size1;
-        if (open(my $cfd, "-|", "cksum", $res->res->json->{temporary})) {
-            ($csum1, $size1) = split(/ /, <$cfd>);
-            close($cfd);
-        }
-        my $csum2 = '2';
-        my $size2;
-        if (open(my $cfd, "-|", "cksum", $file)) {
```

Comment:
\o/ no more NFS!

Reply:
\o/ 🎉 🎉 🎉 🎉

Reply:
careful. we have another use case for NFS: tests reference other/ assets not downloaded into cache.